storagemarket/impl/client.go

```diff
@@ -374,16 +374,16 @@ func (c *Client) ProposeStorageDeal(ctx context.Context, params storagemarket.Pr
 	}
 
 	deal := &storagemarket.ClientDeal{
-		ProposalCid:            proposalNd.Cid(),
-		ClientDealProposal:     *clientDealProposal,
-		State:                  storagemarket.StorageDealUnknown,
-		Miner:                  params.Info.PeerID,
-		MinerWorker:            params.Info.Worker,
-		DataRef:                params.Data,
-		FastRetrieval:          params.FastRetrieval,
-		DealStages:             storagemarket.NewDealStages(),
-		CreationTime:           curTime(),
-		FilestoreCARv2FilePath: params.FilestoreCARv2FilePath,
+		ProposalCid:        proposalNd.Cid(),
+		ClientDealProposal: *clientDealProposal,
+		State:              storagemarket.StorageDealUnknown,
+		Miner:              params.Info.PeerID,
+		MinerWorker:        params.Info.Worker,
+		DataRef:            params.Data,
+		FastRetrieval:      params.FastRetrieval,
+		DealStages:         storagemarket.NewDealStages(),
+		CreationTime:       curTime(),
+		IndexedCAR:         params.IndexedCAR,
```

Comment:
I changed the name of this field to describe function, and not type. I don't think this affects existing deals, because I believe that CBOR is positional, but would love a confirmation, since you've seen several upgrades through @dirkmc

Reply:
I'm not 100% sure about that - maybe write a test to confirm?

Reply:
It's gonna be hard writing a test for that :-( I'll hack something but it won't go in as a test.